WebRoot/css/zm.css

```diff
@@ -6426,70 +6540,74 @@ a.ZmLinkDisabled {
     color: @AppC@;
 }
 
-/* Reading pane mail attachment styles */
-.ZmMailAttachmentsTable {
-	@MailAttachmentsTable@
+/* Attachments styles */
+.ZmAttachmentsTable {
+	@AttachmentsTable@
+}
+
+.ZmAttachmentItem {
+	@AttachmentItem@
 }
 
-.ZmMailAttachmentItem {
-	@MailAttachmentItem@
+.ZmAttachmentItem:hover,
+.ZmAttachmentItem:active,
+.ZmAttachmentItem:focus {
+	@AttachmentItemActive@
 }
 
-.ZmMailAttachmentItem:hover,
-.ZmMailAttachmentItem:active,
-.ZmMailAttachmentItem:focus {
-	@MailAttachmentItemActive@
+.ZmAttachmentIcon,
+.ZmAttachmentIcon svg.svg-icon {
+	@AttachmentIcon@
 }
 
-.ZmMailAttachmentIcon,
-.ZmMailAttachmentIcon svg.svg-icon {
-	@MailAttachmentIcon@
+.ZmAttachmentCheckbox {
+	@AttachmentCheckbox@
 }
 
-.ZmMailAttachmentMetaInfo {
-	@MailAttachmentMetaInfo@
+.ZmAttachmentMetaInfo {
+	@AttachmentMetaInfo@
 }
 
-.ZmMailAttachmentMetaInfo > * {
+.ZmAttachmentMetaInfo > * {
 	display: inline-block;
```

Comment:
can we target more specifically ? just to avoid cases that * may conflict ? maybe use specific tag name? 
Also did we checked that the changes here are not breaking the Calendar app in general ?